package.json

```diff
@@ -3,11 +3,13 @@
   "version": "2.0.0",
   "description": "New Relic log encricher for the `winston` package. Allows `winston` logs to be consumed by New Relic Logs.",
   "main": "index.js",
+  "types": "types/index.d.ts",
   "scripts": {
     "unit": "tap tests/unit/**/*.tap.js --no-coverage",
     "lint": "eslint *.js lib tests",
     "lint:fix": "eslint --fix *.js lib tests",
-    "test": "npm run unit && npm run lint && npm run versioned",
+    "types": "dtslint types",
```

Comment:
Since we have something to validate the types, seems like we should hook that up to CI to auto-verify? Or does that happen as a part of something else?

Reply:
Great, I've added it to the linting workflow.